diff --git a/node_modules/chevrotain/lib/chevrotain.d.ts b/node_modules/chevrotain/lib/chevrotain.d.ts
index 032cd50..8279b78 100644
--- a/node_modules/chevrotain/lib/chevrotain.d.ts
+++ b/node_modules/chevrotain/lib/chevrotain.d.ts
@@ -161,7 +161,7 @@ declare abstract class BaseParser {
    * A Parsing DSL method use to consume a single Token.
    * In EBNF terms this is equivalent to a Terminal.
    *
-   * A Token will be consumed, IFF the next token in the token vector matches <tokType>.
+   * A Token will be consumed, IFF the next token in the token vector matches `tokType`.
    * otherwise the parser may attempt to perform error recovery (if enabled).
    *
    * The index in the method name indicates the unique occurrence of a terminal consumption
